apps/condo/domains/meter/utils/taskSchema/DomaMetersImporter.spec.js

```diff
@@ -70,6 +75,7 @@ describe('DomaMetersImporter', () => {
                 '',
                 '',
                 '',
+                'да',
```

Comment:
You should specify locale explicitly. Right now it's a random test

Reply:
If you run it outside of `ru` region - its failed.
If you run it on device with different local - its failed

Reply:
```
const { Yes } = require('@app/condo/lang/.../')
```

Reply:
I use the same mappers for testing (without lang files).
See class ImporterWrapper at the beginning of the file.

Reply:
Translations is not using in this test. It's possible to use "угу" and "нене" instead.

Reply:
can we replace it with `en` words, since it's the language of open_source.

Or you can generate `yes/no` values (strings) from faker, to test that is working as expected

Reply:
Ok, done